humblegen/src/parser/embeds.rs

```diff
@@ -0,0 +1,89 @@
+use crate::ast::*;
```

Comment:
`*` import again. I believe the other code uses prefixed? I find that a lot more tidy.

Also, a few sentences describing the transform would be nice.

Reply:
```
* import again. I believe the other code uses prefixed? I find that a lot more tidy.
```
Nope, parent mod `parser` does the same thing, and I think that's totally fine given that the parser's job is to construct the AST.